app/src/main/java/org/oppia/android/app/activity/InjectableAutoLocalizedAppCompatActivity.kt

```diff
@@ -13,11 +13,16 @@ import org.oppia.android.app.translation.AppLanguageActivityInjector
 import org.oppia.android.app.translation.AppLanguageActivityInjectorProvider
 import org.oppia.android.app.translation.AppLanguageApplicationInjectorProvider
 
+/**
+ * [Boolean] value that is passed to [AppLanguageWatcherMixin] and should not change for activities inheriting [InjectableAutoLocalizedAppCompatActivity].
+ */
+private const val SHOULD_USE_SYSTEM_LANGUAGE = false
```

Comment:
Is this const actually needed if it's always false? Ditto for the version in InjectableSystemLocalizedAppCompatActivity.